torch_memory_saver/__init__.py

```diff
@@ -14,14 +14,18 @@
 class TorchMemorySaver:
     def __init__(self):
         self._mem_pool = None
+        if os.environ.get("PYTORCH_CUDA_ALLOC_CONF") == "expandable_segments:True":
+            raise RuntimeError(
+                "TorchMemorySaver is disabled for the current process because PYTORCH_CUDA_ALLOC_CONF is set to expandable_segments:True"
```

Comment:
```suggestion
                "TorchMemorySaver is disabled for the current process because expandable_segments is not supported yet (please create an issue if you need it)"
```